packages/scripts/auto-release.sh

```
@@ -0,0 +1,257 @@
+
+decision_of() { cat "$STATE_DIR/$1.decision" 2>/dev/null || true; }
+set_decision() { printf '%s' "$2" > "$STATE_DIR/$1.decision"; }
+version_of() { cat "$STATE_DIR/$1.version" 2>/dev/null || true; }
+set_version() { printf '%s' "$2" > "$STATE_DIR/$1.version"; }
+
+classify_package() {
+  local pkg=$1
+  local paths
+  paths=$(code_paths_for "$pkg")
+  local diff
+  diff=$(cd "packages/$pkg" && git diff "$BASE_REF"...HEAD -- $paths 2>/dev/null || true)
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)